agent/containers/images/Dockerfile.base.j2

```diff
@@ -1,6 +1,6 @@
 {% set is_centos_8 = true if image_name == 'centos' and image_rev == '8' else false %}
 {% set is_centos_9 = true if image_name == 'centos' and image_rev == '9' else false %}
-{% set image_tag = 'stream' + image_rev if image_name == 'centos' else image_rev %}
+{% set image_tag = 'stream' + image_rev if image_name == 'centos' and image_rev != '7' else image_rev %}
```

Comment:
So `image_tag` is `stream8` for CentOS 8 and `stream9` for CentOS 9, but just `7` for CentOS 7?? Or am I missing some magic here? 😦